test/analytics/tests/analytics_from_the_browser.ts

```diff
@@ -172,6 +172,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
       it('should call flush when using the window-exposed flush method', async () => {
         await browser.execute(() => window.__kbnAnalytics.flush());
+        // @ts-ignore-next-line Property 'getFlushAction' does not exist on type '{ getActionsUntilReportTestPluginLifecycleEvent:
```

Comment:
Do you mind to create an issue regarding this so we can investigate what's going on?

Reply:
[kibana/src/plugins/data_views/common/data_views/data_views.test.ts](https://github.com/elastic/kibana/blob/58f49e605bb9ec08205f87f885f8893aa8f0e0fa/src/plugins/data_views/common/data_views/data_views.test.ts#L279) should not be changed. The `result` here is of type: `DataView | void | Error`.

Reply:
@afharo Thanks for finding the other 2 usages.